kubernetes-aws-yaml/Pulumi.yaml

```diff
@@ -0,0 +1,52 @@
+name: k8s-aws-yaml
```

Comment:
I know we have a fix going out soon to stop requiring `.append` files (yay), but for now I think we still need them. (Otherwise when run with `pulumi new`, we'll create a stack with no resources.)

Reply:
Resolved via e084496 (added `Pulumi.yaml.append`)